static/src/javascripts/projects/common/modules/ui/clickstream.js

```diff
@@ -23,104 +22,96 @@ type Spec = {
     customEventProperties?: Object,
 };
 
-type ClickStream = {
-    getClickSpec: (Spec, ?boolean) => Spec | boolean,
-};
+let location;
+let filters;
```

Comment:
Extracting these stateful variables is the only substantive change. The rest of the changes are moving internal functions around